playground/vue-legacy/vite.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'vite'
+import vuePlugin from '@vitejs/plugin-vue'
+import legacyPlugin from '@vitejs/plugin-legacy'
+
+export default defineConfig({
+  base: '',
+  resolve: {
+    alias: {
+      '@': __dirname
+    }
+  },
+  plugins: [
+    legacyPlugin({
+      targets: ['defaults', 'not IE 11', 'chrome > 48']
+    }),
+    vuePlugin()
+  ],
+  build: {
+    minify: false
+  }
```

Comment:
I think this is necessary to test the legacy code.
https://github.com/vitejs/vite/blob/9e9cd23763c96020cdf1807334368e038f61ad01/playground/legacy/vite.config.js#L33-L43